documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -353,20 +379,19 @@ local template = grafana.template;
       .addRow(
         row.new('Samples')
         .addPanel(samplesRate)
+        .addPanel(pendingSamples)
       )
       .addRow(
         row.new(
-          'Shards'
+          'Queue [$url]',
+          repeat='url'
```

Comment:
There could be multiple remote write configs for the same URL, we probably want to repeat on `remote_name:url`, or maybe also include instance. I don't know if we'll be able to get something that works 100% of the time because of HA setups but it's worth trying some other label combinations here.

Reply:
Hmmm after some quick searching it sounds like repeating on multiple vars is not possible. This may end up being best implemented as a table panel.

Reply:
It should be possible with the new Stat panel.

Reply:
@roidelapluie you mean repeating on multiple variables or repeating the panel itself? We could repeat the panel (on `remote_name`) inside the row but that leads to each row having multiple "empty" panels for any `url`/`remote_name` pair that does not match.

It ends up looking like this (wasting a lot of space):
![image](https://user-images.githubusercontent.com/1291846/119793412-da51ba00-bed6-11eb-8b9e-e1a31750c853.png)

Unfortunately, hiding empty panels is currently an open issue in Grafana https://github.com/grafana/grafana/issues/4470.

Reply:
No, I mean using the Stat panel, which supports multiple values.

Reply:
That panel: https://grafana.com/docs/grafana/latest/panels/visualizations/stat-panel/

Reply:
Even with the new stats panel I only see the option to repeat using a single variable:

![screenshot of the repeat options of the new Grafana stats panels](https://user-images.githubusercontent.com/1291846/122745245-17685c80-d289-11eb-8401-af22ec74d2b1.png)

I don't see any of selecting multiple variables on that dropdown.

Reply:
Yeah I don't think it's possible to repeat on more than one label. Table is probably the easiest option unless you want to ask around in some of the Grafana communities discussion spots.